src/mongo/MongoScrapbook.ts

```diff
@@ -181,7 +181,10 @@ class MongoScriptDocumentVisitor extends MongoVisitor<MongoCommand[]> {
 				if (!lastCommand.arguments) {
```

Comment:
This code already existed, but I don't understand how lastCommand is guaranteed to exist.